sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -169,6 +443,12 @@ export enum KnownErrorCode {
     Unauthorized = "Unauthorized"
 }
 
+// @public
+export enum KnownFhirVersion {
+    // (undocumented)
+    Four01 = "4.0.1"
```

Comment:
I'm not in love with `Four01` as a key name. Instead of having an enum for this could we just have a readonly array of known version strings or somesuch?

Reply:
I am worried that indexing into the array may not be the best version discovering experience. I added a new enum with `4.0.1` as a key.